deepmd/pt/model/descriptor/se_a_old.py

```
@@ -0,0 +1,713 @@
+            "spin": None,
+        }
+
+    @classmethod
+    def deserialize(cls, data: dict) -> "DescrptSeA":
+        data = data.copy()
+        check_version_compatibility(data.pop("@version", 1), 2, 1)
+        data.pop("@class", None)
+        data.pop("type", None)
+        variables = data.pop("@variables")
+        embeddings = data.pop("embeddings")
+        env_mat = data.pop("env_mat")
```

Ruff

Local variable `env_mat` is assigned to but never used

Remove assignment to unused variable `env_mat`

(F841)